src/modules/dom-menupulator.js

```diff
@@ -0,0 +1,152 @@
+    if (todo.completed) {
+      this.descriptionSpan.style.textDecoration = 'line-through';
+    } else {
+      this.descriptionSpan.style.textDecoration = 'noemal';
+    }
+
+    // Edit input
+    this.editInput = document.createElement('input');
+    this.editInput.type = 'text';
+    this.editInput.classList.add('edit-input');
+
+    this.infoDiv.appendChild(this.checkIcon);
+    this.infoDiv.appendChild(this.markIcon);
+    this.infoDiv.appendChild(this.descriptionSpan);
+    this.infoDiv.appendChild(this.editInput);
+
+    // ellips icon
+    this.ctaDiv = document.createElement('div');
+    this.ctaDiv.classList.add('action-cta');
+
+    this.ellipsIcon = document.createElement('i');
+    this.ellipsIcon.className = 'fa-solid fa-ellipsis-vertical ellipsis-icon';
+
+    // trush icon
+    this.trushIcon = document.createElement('span');
+    this.trushIcon.className = 'trush-icon';
+    this.tr = document.createElement('i');
+    this.tr.className = 'fa-regular fa-trash-can';
+    this.trushIcon.appendChild(this.tr);
+    this.trushIcon.addEventListener('click', this.delet.bind({ li: this.li, todo }));
+
+    this.ctaDiv.appendChild(this.ellipsIcon);
+    this.ctaDiv.appendChild(this.trushIcon);
+
+    this.li.appendChild(this.infoDiv);
+    this.li.appendChild(this.ctaDiv);
+
+    return this.li;
+  }
+
+  static enableEditing(e) {
+    if (this.editing) {
+      return;
+    }
+    e.stopPropagation();
+    this.editing = true;
+    this.li.classList.add('item-editing');
+    this.infoDiv = this.li.children[0].children;
+    this.ctaDiv = this.li.children[1].children;
+
+    this.infoDiv[2].style.display = 'none';
+    this.ctaDiv[0].style.display = 'none';
+
+    this.infoDiv[3].style.display = 'block';
+    this.infoDiv[3].focus();
+    this.ctaDiv[1].style.display = 'block';
+
+    const val = this.infoDiv[2].innerText;
+    this.infoDiv[3].value = val;
+
+    this.infoDiv[3].addEventListener('keydown', (e) => {
+      if (e.keyCode === 13) {
+        const newValue = e.target.value;
+        this.todo.description = newValue;
+        this.infoDiv[2].innerText = newValue;
+        this.li.classList.remove('item-editing');
+        this.infoDiv[2].style.display = 'block';
+        this.ctaDiv[0].style.display = 'block';
+        DomMenupulator.todoList.edit(this.todo);
+        this.infoDiv[3].style.display = 'none';
+        this.ctaDiv[1].style.display = 'none';
+        this.editing = false;
+      }
+    });
+  }
+
+  static delet(e) {
+    e.stopPropagation();
+    DomMenupulator.todoList.todosContainer.removeChild(this.li);
+    DomMenupulator.todoList.remove(this.todo);
+  }
+
+  static markAsCopleted(e) {
+    e.stopPropagation();
+    this.infoDiv = this.li.children[0].children;
+    this.infoDiv[0].style.display = 'none';
+    this.infoDiv[1].style.display = 'block';
+    this.infoDiv[2].style.textDecoration = 'line-through';
+
+    DomMenupulator.todoList.complete(this.todo);
+  }
+}
```

Comment:
Hi Mr. me, 
you have used some hardcoded `indexes` numbers, which might make a huge issue, when changing the code, instead of that you loop throughout it and some logic for that.